op-guide/configuration.md

```diff
@@ -22,6 +22,11 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + default: "info"
 + You can choose from debug, info, warn, error, or fatal.
 
+### --log-file
++ the log file
++ default: ""
++ If this flag is not set, log will be written to stdout. Otherwise log will be stored in the log file. And log file will be automatically rotated by day.
```

Comment:
"Write" seems ok with me.
About the last part, is it okay if we change it to "Otherwise, logs will be stored in the log file which will be automatically rotated every day."